test/integration/fixtures/test-image-repository-mapping.json

```diff
@@ -1,98 +1,66 @@
 {
-  "apiVersion": "v1beta1",
+  "kind": "ImageStreamMapping",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "test",
+    "creationTimestamp": null
+  },
   "image": {
+    "metadata": {
+      "name": "abcd1234",
+      "creationTimestamp": null
+    },
+    "dockerImageReference": "openshift/ruby-19-centos:latest",
```

Comment:
I know this is being picky :stuck_out_tongue:  , but we don't support this image, can we then use ruby-20-centos7 or some random image name here, pretty please?